### PR TITLE
docs: update readme with multiple scope syntax

### PR DESCRIPTION
**Pull request by MichaelGoberling**
## Description

Update the README with the required syntax for multiple scopes

## Related Issue

https://github.com/adobe/aio-apps-action/issues/39

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by fe-lix- — APPROVED** [review]
LGTM.

Alternative phrasing:

```Comma separated list of required meta scopes for JWT token. ```

It has confused some to see the mention of scopes and meta-scopes. The error messages speaks about metascopes.

File: README.md [hunk 1]
```diff
@@ -43,8 +43,9 @@ This Github action supports following commands
       2) CLIENTSECRET - Client secret for the Adobe I/O console project
       3) TECHNICALACCOUNTID - Technical account Id for the Adobe I/O console project
       4) IMSORGID - IMS Org Id
-      5) SCOPES - Comma separated list of required scopes for JWT token.
-      6) KEY - Private key associated with project
+      5) SCOPES - Bracket-enclosed, double-quoted, and comma-separated list of required scopes for JWT token
+          - Example: `["scope1", "scope2"]`
+      7) KEY - Private key associated with project
```
Comment (fe-lix-) [inline README.md:48]: It should still be a 6

Comment (MichaelGoberling) [reply README.md]: Nice catch. Fixed

**Comment by MichaelGoberling** [comment]
@fe-lix- 

Good point. Updated the README to reference "meta scopes"

**Pull request merged**